server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/job/SqlPlanJob.java

```diff
@@ -53,9 +54,11 @@ public void execute(JobExecutionContext context) {
             }
         }
 
+        DatabaseChangeParameters taskParameters = JSON.parseObject(scheduleEntity.getJobParametersJson(),
```

Comment:
avoid refer fastjson, use jackson instead

Reply:
This was originally using fastjson, sorry for I didn't fix it. Corrected